content/javascript/concepts/arrays/terms/indexOf/indexOf.md

```diff
@@ -27,13 +27,13 @@ A negative `startIndex` will offset from the end of the array to begin search.
 
 ## Examples
 
-Find the index of `2` in the `numbers` array:
+In the example below, the index with a value of 12 is logged to the console:
```

Comment:
```suggestion
In the example below, the index for the number 12 is logged to the console:
```

Reply:
Not sure my suggestion is ideal but the original is confusing to my ear. It suggests to me that the index = 12 as opposed to the index of the value 12 in the list